test/test_project/settings.py

```diff
@@ -11,6 +11,7 @@ def project_path(path):
     return abspath(join(dirname(__file__), path))
 
 
+sys.path.insert(0, project_path('.'))
```

Comment:
There's no need to hack the Python path. Just refer to the test app by its full module path: `test_project.test_app`.